Examples/Lenses/AutoLenses.stencil

```diff
@@ -0,0 +1,38 @@
+infix operator *~: MultiplicationPrecedence
```

Comment:
I know narf personally :) @bejar37 I'd be happy to add your version as an example if you want to contribute